### ISSUE TITLE
[ANOVA] ANCOVA

### ISSUE THREAD

**Issue by TimKDJ** [TimKDJ/jasp-desktop#56]
Commit messages since the previous release: 

```
 Speed up An(c)ova and .v() (#4244)

* more intelligent testing of groups

* .v only once in .applyOnGroups + faster lookup of encode/ decode function

* fix unit tests

* implement feedback

RM ANOVA plot: apply within-participants normalization for CI descriptives (#4206)

* Descriptive plot RM ANOVA uses withinSE now

* apply within-participants normalization for CI's descriptives

* Added unit tests for descriptives plots

* Speed up ANOVA 
```

 You can find information about the testing procedure [here](https://github.com/jasp-stats/jasp-test-release/blob/master/README.md).
Please do not post the bugs you find in this issue, but open a new issue for each bug.
